### PR TITLE
modify to add http service guide

### PR DESCRIPTION
**Pull request by KimiLiuQh**
add the python version of http service example in cluster serving, and way to parse requested json data.

### CODE REVIEW

File: docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb [hunk 1]
```diff
@@ -678,6 +678,154 @@
...
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def parse_json_to_ndarray(r):\n",
```
Comment (glorysdj) [inline docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:739]: please move this to cluster serving api and code here to call the api
add please also refine the doc

Comment (glorysdj) [reply docs/docs/ClusterServingGuide/OtherFrameworkUsers/keras-to-cluster-serving-example.ipynb:739]: https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/zoo/serving/client.py

**Comment by Litchilitchy** [comment]
@KimiLiuQh This could be renamed to `cluster-serving-http-example` and there would be no conflict and can merge.

**Comment by Litchilitchy** [comment]
I would merge this and open another to update it according to latest changes.

**Pull request merged**